scripts/create_eks_cluster.sh

```diff
@@ -0,0 +1,261 @@
AuL2RwZGsvbWVzb24uYnVpbGQKCXNlZCAtaSAicy9zdWJkaXIoJ2RyaXZlcnMnKS8jc3ViZGlyKCdkcml2ZXJzJykvZyIgLi9kcGRrL21lc29uLmJ1aWxkCglzZWQgLWkgInMvc3ViZGlyKCd1c2VydG9vbHMnKS8jc3ViZGlyKCd1c2VydG9vbHMnKS9nIiAuL2RwZGsvbWVzb24uYnVpbGQKCXNlZCAtaSAicy9zdWJkaXIoJ2FwcCcpLyNzdWJkaXIoJ2FwcCcpL2ciIC4vZHBkay9tZXNvbi5idWlsZAoJc2VkIC1pICJzL3N1YmRpcignZG9jJykvI3N1YmRpcignZG9jJykvZyIgLi9kcGRrL21lc29uLmJ1aWxkCglzZWQgLWkgInMvc3ViZGlyKCdleGFtcGxlcycpLyNzdWJkaXIoJ2V4YW1wbGVzJykvZyIgLi9kcGRrL21lc29uLmJ1aWxkCglzZWQgLWkgInMvaW5zdGFsbF9zdWJkaXIoJ2V4YW1wbGVzJywvI2luc3RhbGxfc3ViZGlyKCdleGFtcGxlcycsL2ciIC4vZHBkay9tZXNvbi5idWlsZAoJc2VkIC1pICJzQGluc3RhbGxfZGlyOiBnZXRfb3B0aW9uKCdkYXRhZGlyJylAI2luc3RhbGxfZGlyOiBnZXRfb3B0aW9uKCdkYXRhZGlyJylAZyIgLi9kcGRrL21lc29uLmJ1aWxkCglzZWQgLWkgInMvZXhjbHVkZV9maWxlczogJ21lc29uLmJ1aWxkJykvI2V4Y2x1ZGVfZmlsZXM6ICdtZXNvbi5idWlsZCcpL2ciIC4vZHBkay9tZXNvbi5idWlsZAoKCSMjIyMjIyMjIyMgUEFUQ0hJTkcgRFBESyAjIyMjIyMjIyMjCgoJY2QgLi9kcGRrCgltZXNvbiBidWlsZCAtRGVuYWJsZV9rbW9kcz10cnVlIC1Ea2VybmVsX2Rpcj0vbGliL21vZHVsZXMvJCh1bmFtZSAtcikvCgluaW5qYSAtQyBidWlsZAoKCXN1ZG8gbXYgLi9idWlsZC9rZXJuZWwvbGludXgvaWdiX3Vpby9pZ2JfdWlvLmtvICR7SUdCX1VJT19QQVRIfQoJc3VkbyBjaG93biByb290OnJvb3QgJHtJR0JfVUlPX1BBVEh9Cn0KCmNvbmZpZ3VyZV9kcGRrX2ludGVycnVwdF9tb2RlX3N1cHBvcnQgKCkKewoJIyBkb3dubG9hZCBhbmQgYnVpbGQgYW5kIGluc3RhbGwgdGhlIHZmaW8tcGNpIGRyaXZlciB3aXRoIHdjIHN1cHBvcnQKCSMgZm9yIEVOQXYyCgljZCAkQlVJTERfRElSCglnaXQgY2xvbmUgaHR0cHM6Ly9naXRodWIuY29tL2Ftem4vYW16bi1kcml2ZXJzLmdpdAoJY2QgYW16bi1kcml2ZXJzL3VzZXJzcGFjZS9kcGRrL2VuYXYyLXZmaW8tcGF0Y2gKCS4vZ2V0LXZmaW8td2l0aC13Yy5zaAoKCSMgTk9URTogdXNlIHN1ZG8gd2hlbi9pZiBydW5uaW5nIHRoZSBzY3JpcHQgbWFudWFsbHkKCSMgbG9hZCB0aGUgZHJpdmVyCgltb2Rwcm9iZSB2ZmlvLXBjaQoKCSMgZW5hYmxlIHVuc2FmZV9ub2lvbW11X21vZGUKCWVjaG8gMSA+IC9zeXMvbW9kdWxlL3ZmaW8vcGFyYW1ldGVycy9lbmFibGVfdW5zYWZlX25vaW9tbXVfbW9kZQoKCSMgcGVyc2lzdCB0aGUgY2hhbmdlcyBhY3Jvc3MgcmVib290cwoJY2F0IDw8IEVPRiA+PiAvZXRjL3JjLmQvcmMubG9jYWwKbW9kcHJvYmUgdmZpby1wY2kKZWNobyAxID4gL3N5cy9tb2R1bGUvdmZpby9wYXJhbWV0ZXJzL2VuYWJsZV91bnNhZmVfbm9pb21tdV9tb2RlCkVPRgoJY2htb2QgK3ggL2V0Yy9yYy5kL3JjLmxvY2FsCgoJcm0gLXJmICRCVUlMRF9ESVIKfQoKY29uZmlndXJlX21hY2hpbmUgKCkKewoJc3VkbyBybSAtZiAvZXRjL2NuaS9uZXQuZC8xMC1hd3MuY29uZmxpc3QKCXN1ZG8gbW9kcHJvYmUgdWlvCglpZiBbIHgkKGxzbW9kIHwgYXdrICd7IHByaW50ICQxIH0nIHwgZ3JlcCBpZ2JfdWlvKSA9PSB4IF07IHRoZW4KCQlidWlsZF9hbmRfaW5zdGFsbF9pZ2JfdWlvCgkJc3VkbyBpbnNtb2QgL2xpYi9tb2R1bGVzLyQodW5hbWUgLXIpL2tlcm5lbC9kcml2ZXJzL3Vpby9pZ2JfdWlvLmtvIHdjX2FjdGl2YXRlPTEKCWZpCgoJIyBjb25maWd1cmUgaHVnZXBhZ2VzIGFuZCBwZXJzaXN0IHRoZSBjb25maWcgYWNyb3NzIHJlYm9vdHMKCXN1ZG8gc3lzY3RsIC13IHZtLm5yX2h1Z2VwYWdlcz0ke0hVR0VQQUdFU30KCWlmIFsgLWYgL3N5cy9mcy9jZ3JvdXAvaHVnZXRsYi9rdWJlcG9kcy9odWdldGxiLjJNQi5saW1pdF9pbl9ieXRlcyBdOyB0aGVuCgkJZWNobyAkKChIVUdFUEFHRVMgKiAyICogMTAyNCAqIDEwMjQpKSB8IHRlZSAvc3lzL2ZzL2Nncm91cC9odWdldGxiL2t1YmVwb2RzL2h1Z2V0bGIuMk1CLmxpbWl0X2luX2J5dGVzCglmaQoJZWNobyAidm0ubnJfaHVnZXBhZ2VzPSR7SFVHRVBBR0VTfSIgPj4gL2V0Yy9zeXNjdGwuY29uZgp9Cgpjb25maWd1cmVfbWFjaGluZQpjb25maWd1cmVfZHBka19pbnRlcnJ1cHRfbW9kZV9zdXBwb3J0CgotLT09TVlCT1VOREFSWT09LS0K",
```

Comment:
This blob is pretty scary. Could we re-generate it from source at runtime? Or at least we should add a comment indicating to users how this is generated, so that they can verify the contents if they want.

Reply:
Agree about the scary part. Script now generates the stuff at runtime and also added comments explaining what the script does and why.